pandas/tests/series/methods/test_isin.py

```diff
@@ -234,3 +234,28 @@ def test_isin_filtering_with_mixed_object_types(data, is_in):
     expected = Series([True, False])
 
     tm.assert_series_equal(result, expected)
+
+
+@pytest.mark.parametrize(
+    "data,is_in,expected",
```

Comment:
looks like `is_in` and `expected` are the same for the two test cases - if so, perhaps just define them in the test body, and only parametrize over `data`?